build/azure-pipelines.yml

```diff
@@ -14,6 +14,10 @@ pool:
 
 variables:
   buildConfiguration: 'Release'
+  major: $(MajorVersion)
+  minor: $(MinorVersion)
+  patch: $[counter(variables['minor'], 8)]
+  versionNumber: $(major).$(minor).$(patch)
 
 steps:
 - task: NuGetToolInstaller@1
```

Comment:
Since we need to use `dotnet pack` can we use `dotnet restore` and get rid of `NuGetToolInstaller`?

Reply:
Correct. Will be implemented in future revisions of build pipe configuration.